apigee_edge.module

```diff
@@ -1081,6 +1081,10 @@ function template_preprocess_app_credential(array &$variables) {
       ];
     }
     elseif ($def['value_type'] === 'status') {
+      // Check if expired.
+      if ($normalized['expiresAt'] !== -1 && \Drupal::time()->getRequestTime() - intval($normalized['expiresAt'] / 1000) > 0) {
```

Comment:
You should set the "Expired" special value for the `expiredAt` field earlier, here:

```php
<?php
    if ($def['value_type'] == 'date') {
      // TODO Should we make format configurable?
      /** @var \DateTimeInterface $value */
      if ($value !== -1) {
        $time_diff = \Drupal::time()->getRequestTime() - intval($value / 1000);
        if ($time_diff > 0) {
          $value = t('@time ago', ['@time' => $dateFormatter->formatTimeDiffSince(intval($value / 1000))]);
        }
        else {
          $value = t('@time hence', ['@time' => $dateFormatter->formatTimeDiffUntil((int) ($value / 1000))]);
        }
      }
      else {
        $value = t('Never');
      }
    }

```

Reply:
There was no comment here so I was not sure if this code is to display the expired date only or can `$value` be used here.

Wouldn't this `if ($def['value_type'] == 'date') {` be TRUE for `issuedAt` and `expiresAt` only?